contracts/contracts/MACI.sol

```diff
@@ -188,15 +188,14 @@ contract MACI is IMACI, DomainObjs, Params, Utilities {
       revert InvalidPubKey();
     }
 
-    MaxValues memory maxValues = MaxValues({
-      maxMessages: uint256(MESSAGE_TREE_ARITY) ** _treeDepths.messageTreeDepth,
-      maxVoteOptions: uint256(MESSAGE_TREE_ARITY) ** _treeDepths.voteOptionTreeDepth
-    });
+    MaxValues memory maxValues = MaxValues({ maxMessages: 1000, maxVoteOptions: 25 });
+
+    BatchSizes memory batchSizes = BatchSizes({ messageBatchSize: MESSAGE_BATCH_SIZE });
```

Comment:
no need to have a struct for one value, need to set this as an immutable variable for the poll contract